pyproject.toml

```
@@ -1,8 +1,46 @@
+[project]
+name = "hubdocs"
+description = "Documentation for the Hubverse collection of collaborative modeling tools"
+authors = [
+    {name = "Consortium of Infectious Disease Modeling Hubs"},
+    {name = "Graziella", email = "graziella@lumache.com"}
```

I think Graziella was from the readthedocs template. I don't know if she is real. 

```suggestion
    {name = "Consortium of Infectious Disease Modeling Hubs"}
```

---

I was wondering who that was 😂